docs/paper.bib

```diff
@@ -0,0 +1,361 @@
+  title={Circular Flip-Flop Index: quantifying revision stability of forecasts of direction},
+  author={Griffiths, Deryn and Loveday, Nicholas and Price, Benjamin and Foley, Michael and McKelvie, Alistair},
+  journal={Journal of Southern Hemisphere Earth Systems Science},
+  volume={71},
+  number={3},
+  pages={266--271},
+  year={2021},
+  publisher={CSIRO Publishing},
+  doi={10.1071/es21010}
+}
+@misc{loveday2024jive,
+      title={The Jive Verification System and its Transformative Impact on Weather Forecasting Operations}, 
+      author={Nicholas Loveday and Deryn Griffiths and Tennessee Leeuwenburg and Robert Taggart and Thomas C. Pagano and George Cheng and Kevin Plastow and Elizabeth Ebert and Cassandra Templeton and Maree Carroll and Mohammadreza Khanarmuei and Isha Nagpal},
+      year={2024},
+      eprint={2404.18429},
+      archivePrefix={arXiv},
+      primaryClass={physics.ao-ph},
+	  doi={10.48550/arXiv.2404.18429}
+}
+@article{Ferro_2013, 
+	title={Fair scores for ensemble forecasts: Fair Scores for Ensemble Forecasts}, 
+	volume={140}, 
+	ISSN={0035-9009}, 
+	url={http://dx.doi.org/10.1002/qj.2270}, 
+	DOI={10.1002/qj.2270}, 
+	number={683}, 
+	journal={Quarterly Journal of the Royal Meteorological Society}, 
+	publisher={Wiley}, 
+	author={Ferro, C. A. T.}, 
+	year={2013}, 
+	month=dec, 
+	pages={1917–1923}
+}
+@article{Gneiting_2007, 
+	title={Strictly Proper Scoring Rules, Prediction, and Estimation}, 
+	volume={102}, 
+	ISSN={1537-274X}, 
+	url={http://dx.doi.org/10.1198/016214506000001437}, 
+	DOI={10.1198/016214506000001437}, 
+	number={477}, 
+	journal={Journal of the American Statistical Association}, 
+	publisher={Informa UK Limited}, 
+	author={Gneiting, Tilmann and Raftery, Adrian E}, 
+	year={2007}, 
+	month=mar, 
+	pages={359–378}
+}
+@article{gmd-12-4185-2019,
+	author = {Pulkkinen, S. and Nerini, D. and P\'erez Hortal, A. A. and Velasco-Forero, C. and Seed, A. and Germann, U. and Foresti, L.},
+	title = {Pysteps: an open-source Python library  for probabilistic precipitation nowcasting (v1.0)},
+	JOURNAL = {Geoscientific Model Development},
+	VOLUME = {12},
+	YEAR = {2019},
+	NUMBER = {10},
+	PAGES = {4185--4219},
+	URL = {https://gmd.copernicus.org/articles/12/4185/2019/},
+	DOI = {10.5194/gmd-12-4185-2019}
+}
+@article{Imhoff:2023,
+	author = {Imhoff, Ruben O. and De Cruz, Lesley and Dewettinck, Wout and Brauer, Claudia C. and Uijlenhoet, Remko and van Heeringen, Klaas-Jan and Velasco-Forero, Carlos and Nerini, Daniele and Van Ginderachter, Michiel and Weerts, Albrecht H.},
+	title = {Scale-dependent blending of ensemble rainfall nowcasts and numerical weather prediction in the open-source pysteps library},
+	journal = {Quarterly Journal of the Royal Meteorological Society},
+	volume = {149},
+	number = {753},
+	pages = {1335-1364},
+	keywords = {blending, early warning, nowcasting, numerical weather prediction, open source, rainfall},
+	doi = {https://doi.org/10.1002/qj.4461},
```

Comment:
```suggestion
	doi = {10.1002/qj.4461},
```
Need to reformat (as suggested above) so that it renders properly in the JOSS PDF